namespace cpp dsn

struct error_code
{
    1: string code;
}

struct task_code
{
    1: string code;
}